eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -56,8 +56,8 @@ jobs:
 
     # Linux musl arm64
     - ${{ if and(eq(parameters.platform, 'linux_musl_arm64'), or(eq(parameters.jobParameters.isExtraPlatforms, true), eq(parameters.jobParameters.includeAllPlatforms, true))) }}:
-      - (Alpine.313.Arm64.Open)ubuntu.1804.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13-helix-arm64v8
-      - (Alpine.314.Arm64.Open)ubuntu.1804.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.14-helix-arm64v8
+      - (Alpine.317.Arm64.Open)ubuntu.1804.armarch.open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.17-helix-arm64v8
```

Comment:
There's a inclusion of 3.15 and 3.17 few lines above. I'm trying to decode the conditions and difference between them:
```
above:
linux_musl_x64 && ( ! isExtraPlatforms || includeAllPlatforms) --> 3.15
linux_musl_x64 && (   isExtraPlatforms || includeAllPlatforms) --> 3.17

here:
linux_musl_x64 && (   isExtraPlatforms || includeAllPlatforms) --> 3.15 && 3.17
```

Should this be somehow collapsed into one block? Are we running twice on 3.17 thanks to this?

Reply:
I probably don't understand the comment.  the section above (assuming lines 50-55) are for x86 e.g. amd64. Line 59 is for arm64 e.g. arm64v8

Reply:
Fudge, `arm` and `amd` look too similar :rofl: Nevermind then.